Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -132,6 +134,8 @@ def parse_arguments(argv):
             config.config_path = opt[1]
         if opt[0] == "-k" or opt[0] == "--kubeconfig":
             config.kubeconfig = opt[1]
+        if opt[0] == "--context":
```

Comment:
```suggestion
        if opt[0] == "-C" or opt[0] == "--context":
```

If you agree that a shorthand should be available.